man/man1/exiv2.1

```diff
@@ -76,6 +77,21 @@ image are determined.
 .IP \(bu 2
 Reading other TIFF-like RAW image formats, which are not listed in the
 table, may also work.
+.IP \(bu 2
+Support for BMFF-based types such as AVIF, CR3, HEIF, HEIC and JXL is 
+a build option (see https://github.com/Exiv2/exiv2/blob/main/README.md#BMFF). 
```

Comment:
I'd remove this link as it is transient (or use tag/branch specific address behind e.g. word "here" instead)

Reply:
> I'd remove this link as it is transient (or use tag/branch specific address behind e.g. word "here" instead)

Yes, this helps with maintenance.